src/FilterList.js

```diff
@@ -206,6 +189,23 @@ let FilterList = _.flow(
                       ...mapNodeToProps(child, fields),
                     }}
                   />
+                  {!child.updating &&
+                    tree.disableAutoUpdate &&
+                    // find if any nodes in the tree are marked for update (i.e. usually nodes are marked for update because they react to "others" reactor)
+                    _.some(
+                      treeNode => treeNode !== node && treeNode.markedForUpdate,
+                      F.treeToArray(_.get('children'))(tree.tree)
+                    ) && (
+                      <div
+                        className="apply-filter-button"
+                        onClick={e => {
+                          e.stopPropagation()
+                          tree.triggerUpdate()
+                        }}
+                      >
+                        <Button primary>APPLY FILTER</Button>
```

Comment:
the gv styles should make this all caps, but the source should probably have Title Case